### PR TITLE
Add integration test instruction in PR requirements

### PR DESCRIPTION
**Pull request by lyuyuan92**
#1200 

Add integration test instruction in the Pull Request requirements list.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ Patches should be submitted as [pull requests](https://help.github.com/articles/
 Before submitting a PR:
 - Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed.
   - Run `cargo test --all` to make sure all tests still pass and no warnings are emitted.
+  - Run `cargo run -p sync-test` to make sure the integration sync test still pass.
```
Comment (thomcc) [inline docs/contributing.md]:
```suggestion
  - Run `cargo run -p sync-test` to make sure the sync integration tests still pass.
```

File: docs/contributing.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ Patches should be submitted as [pull requests](https://help.github.com/articles/
...
+  You need to have [node](https://nodejs.org/en/) installed for running the integration test.
```
Comment (thomcc) [inline docs/contributing.md]:
```suggestion
  Note: You need to have [node](https://nodejs.org/) installed for running the integration tests.
```

Comment (vladikoff) [inline docs/contributing.md]: for l10n the url can be just https://nodejs.org

Comment (lyuyuan92) [reply docs/contributing.md]: Done

Comment (lyuyuan92) [reply docs/contributing.md]: Done

**Pull request merged**